lib/segment/src/index/visited_pool.rs

```diff
@@ -14,52 +14,81 @@ const POOL_KEEP_LIMIT: usize = 16;
 /// It stores the sequence number of last processed operation next to the point ID, which allows to avoid memory allocation
 /// and reuse same counter for multiple queries.
```

Comment:
This doc comment should be updated as well (or moved to `VisitedListData`).

Reply:
You are right

Reply:
Fixed